lib/fileUpload.coffee

```diff
@@ -13,7 +13,14 @@ if UploadFS?
 		cookie = new Cookies()
 		if Meteor.isClient
 			document.cookie = 'rc_uid=' + escape(Meteor.userId()) + '; path=/'
-			document.cookie = 'rc_token=' + escape(Meteor._localStorage.getItem('Meteor.loginToken')) + '; path=/'
+			pathPrefix = __meteor_runtime_config__.ROOT_URL_PATH_PREFIX
+			loginTokenKey = 'Meteor.loginToken'
+
+			# account for if being run in subdirectory
+			if pathPrefix
+				document.cookie = 'rc_token=' + escape(Meteor._localStorage.getItem(loginTokenKey + ':/:' + pathPrefix)) + '; path=/'
+			else
+				document.cookie = 'rc_token=' + escape(Meteor._localStorage.getItem(loginTokenKey)) + '; path=/'
```

Comment:
We can improve all this part using `Accounts._storedLoginToken()` to get the token, what do you think? This method uses `Accounts.LOGIN_TOKEN_KEY` that already have the correct key, so seems to be the best way to do that.